src/cogs/split.py

```diff
@@ -0,0 +1,203 @@
+        """Get the channel id from the settings and construct the VoiceChannel object if one is set."""
+        if self.data.lobby_channel_id:
+            return self.bot.get_channel(self.data.lobby_channel_id)
+
+    @property
+    def team_one_channel(self) -> Optional[VoiceChannel]:
+        """Get the channel id from the settings and construct the VoiceChannel object if one is set."""
+        if self.data.team_one_channel_id:
+            return self.bot.get_channel(self.data.team_one_channel_id)
+
+    @property
+    def team_two_channel(self) -> Optional[VoiceChannel]:
+        """Get the channel id from the settings and construct the VoiceChannel object if one is set."""
+        if self.data.team_two_channel_id:
+            return self.bot.get_channel(self.data.team_two_channel_id)
+
+    @group(aliases=["settings", "conf"])
+    async def config(self, ctx: Context) -> None:
+        """Display the current config."""
+
+        if not ctx.subcommand_passed:
+            embed = Embed(title="SplitTeam Config", colour=EMBED_COLOR)
+
+            lobby = self.lobby_channel
+            embed.add_field(name="Lobby", value=lobby.mention if lobby else "Not Configured")
+
+            team_one = self.team_one_channel
+            embed.add_field(name="Team One", value=team_one.mention if team_one else "Not Configured")
+
+            team_two = self.team_two_channel
+            embed.add_field(name="Team Two", value=team_two.mention if team_two else "Not Configured")
+
+            embed.add_field(name="Minimum Team Size", value=str(self.data.minimum_team_size))
+
+            role = ctx.guild.get_role(self.data.captain_role_id)
+            embed.add_field(name="Captain Role", value=role.mention if role else "Not Configured")
+
+            await ctx.send(embed=embed)
+
+    @config.command()
+    async def lobby(self, ctx: Context, lobby: VoiceChannel) -> None:
+        """Set the lobby channel to be used for splitting."""
+        self.data.lobby_channel_id = lobby.id
+        await ctx.send(f"> Lobby set to {lobby.mention}", delete_after=5)
+
+    @config.command(aliases=["one", "team1", "t1"])
+    async def team_one(self, ctx: Context, channel: VoiceChannel) -> None:
+        """Set the voice channel for team one."""
+        self.data.team_one_channel_id = channel.id
+        await ctx.send(f"> Team One set to {channel.mention}", delete_after=5)
+
+    @config.command(aliases=["two", "team2", "t2"])
+    async def team_two(self, ctx: Context, channel: VoiceChannel) -> None:
+        """Set the voice channel for team two."""
+        self.data.team_two_channel_id = channel.id
+        await ctx.send(f"> Team Two set to {channel.mention}", delete_after=5)
+
+    @lobby.error
+    @team_two.error
+    @team_one.error
+    async def channel_error(self, ctx: Context, error: CommandError) -> None:
+        if isinstance(error, BadArgument):
+            await ctx.send("> Please make sure you are tagging a **Voice Channel**.")
```

Comment:
Other kinds of error are silenced here. Maybe they should be re-raised so that they can be handled elsewhere or logged.
```suggestion
        if isinstance(error, BadArgument):
            await ctx.send("> Please make sure you are tagging a **Voice Channel**.")
        else:
            raise error
```

Reply:
I'm not 100% sure this is the correct way to do it in `discord.py`, fyi 😅